src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1044,6 +1045,31 @@ private bool CheckPropertyValueKind(SyntaxNode node, BoundExpression expr, BindV
             return true;
         }
 
+        private bool IsBadBaseAccess(SyntaxNode node, BoundExpression receiverOpt, MethodSymbol method, DiagnosticBag diagnostics,
+                                     PropertySymbol propertySymbolOpt = null, bool checkOnlyAccessThroughInterface = false)
+        {
+            if (receiverOpt?.Kind == BoundKind.BaseReference)
+            {
+                var baseReference = (BoundBaseReference)receiverOpt;
+
+                if (method.IsImplementableInterfaceMember() &&
```

Comment:
>IsImplementableInterfaceMember [](start = 27, length = 30)

Just as you can use a base invocation to a non-virtual member in a class, base invocations to an interface do not require that the symbol found is virtual or abstract.